### DISCUSSION TITLE
导入 v1 配置失败

### DISCUSSION THREAD

**Discussion by RtYkk** [the1812/Bilibili-Evolved#2750] [提问 / Q&A] [answered]
2.0.8，现在回滚到 v1 了……
[bilibili-evolved-settings.txt](https://github.com/the1812/Bilibili-Evolved/files/7750298/bilibili-evolved-settings.txt)
 导入的时候显示成功，但是刷新之后一个功能也没出来。

**Comment by the1812** [comment, marked as answer]
用你发的设置试了一下, 没问题啊

**Comment by RtYkk** [comment]
麻烦您了…等我明天再试试看。不知道上次什么情况
